utilities/CMakeLists.txt

```diff
@@ -86,3 +86,19 @@ else()
     message("-- ${Green}Utilities -- runvx module added with CPU support${ColourReset}")
     message("-- ${Red}WARNING: Utilities -- runcl, loom_shell, & mv_deploy modules excluded${ColourReset}")
 endif()
+
+if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY)
+  if(FFMPREG_FOUND)
+    #include(ROCMHeaderWrapper)
+    # Copy Header files to build directory
+    configure_file("./mv_deploy/mvdeploy_api.h" "${PROJECT_BINARY_DIR}/include/mivisionx/mvdeploy_api.h" COPYONLY)
```

Comment:
change "${PROJECT_BINARY_DIR}/include/mivisionx/mvdeploy_api.h" to "${PROJECT_BINARY_DIR}/include/mvdeploy_api.h"